GeoHealthCheck/config_main.py

```diff
@@ -37,7 +37,8 @@
 # Replace None with 'your secret key string' in quotes
 SECRET_KEY = None
 
-GHC_RETENTION_DAYS = 30
+# days, seconds, microseconds, milliseconds, minutes, hours, weeks
+GHC_RETENTION_DAYS = (30, 0, 0, 0, 0, 0, 0)
```

Comment:
Try to retain the existing type for `GHC_RETENTION_DAYS`, may expand this to a `timedelta` explicitly.